README.md

```diff
@@ -10,3 +10,12 @@ This repository contains
      * configuration file parsing
      * argument parsing
      * RPCs
+  3. The following CLI tools for debugging:
+     * lib/channel_helper.exe -- a channel passing helper CLI
+     * [A] lib_test/test.exe -- a test suite built from tests in lib_test
+     * [A] example/example.exe -- executable of example Xcp code
+     * memory/memory_cli.exe -- a squeezed debugging CLI
+     * v6/v6_cli.exe -- a V6d debugging CLI
+
+      [A] - `make test` automatically builds and runs this
```

Comment:
I think we don't need to mention this

Reply:
I don't think this info is included anywhere else, since `INSTALL.md` just points to the `README.md` for "detailed build instructions". Might be worth including for the sake of completion?

Reply:
(This would be redundant if we removed references to `test` and `example` from the list of CLI tools.)

Reply:
Actually, is it worth expanding on `make test` in `INSTALL.md`?

Reply:
It's outdated, I think we should remove it

Reply:
Remove `INSTALL.md` or discussion of `make test`?